.github/workflows/dispatch_release.yml

```diff
@@ -1,31 +0,0 @@
-### The auto release workflow triggered through dispatch request from CI
-name: auto-release
-
```

Comment:
Do we not need releases for zstream?

Reply:
We dont do releases for zStream only master since we are mostly interested in Framework changes and they are being done in master and cherrypicked so does not make sense to have zReleases !